### PR TITLE
fixed the equal function

### PR DESCRIPTION
**Pull request by MalcolmBarbee**

### CODE REVIEW

File: src/containers/calculator.js [hunk 1]
```diff
@@ -43,29 +31,22 @@ class Calculator extends React.Component {
         const num = a / b
         return num;
     }
-    percent = (num) => {
+    percent = (num, previous = 1) => {
         return num / 100;
```
Comment (FiveEightyEight) [inline src/containers/calculator.js]: this should be `return (previous * num) / 100;`

File: src/containers/calculator.js [hunk 2]
```diff
@@ -107,8 +91,15 @@ class Calculator extends React.Component {
         if(!operate && value === 'equal') {
             return;
         }
-
-
+        if(value === 'percent') {
+            const num = this.percent(display).toString();
+            this.setState ({
+                displayValue: num,
+                operation: value,
```
Comment (FiveEightyEight) [inline src/containers/calculator.js]: Remove this line:
`operation: value,`

**Review by FiveEightyEight — CHANGES_REQUESTED** [review]
Two changes needed

File: src/containers/calculator.js (hunk 1 above)
Comment (MalcolmBarbee) [reply src/containers/calculator.js]: Got it

**Review by FiveEightyEight — APPROVED** [review]

**Pull request merged**